src/components/Banner/ArticleBanner.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Banner from './Banner';
+import { ContainedButton } from './Banner.style';
+import { Link } from 'react-router-dom';
+import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
+
+export const BANNER_COPY =
+  'The U.S. is in a third COVID wave. Exercise extra caution in order to reduce your risk of infection, to avoid infecting others, and to prevent an overwhelming of our healthcare systems.';
+
+const trackClick = () => {
+  trackEvent(EventCategory.ARTICLES, EventAction.CLICK, 'Banner: Third wave');
+};
+
+// TODO: Update with the URL to the article
+const articleURL = '/';
+
+const renderButton = () => (
```

Comment:
looks like this is unused?

Reply:
I'm leaving it, because we should be adding the button soon (if I comment it out, I need to comment out the imports as well, it would be annoying)